### PR TITLE
Add EIA 860 M into zenodo store

### PR DESCRIPTION
**Pull request by cmgosnell**
wahoo!

### CODE REVIEW

File: frictionless/eia860m_raw.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+def datapackager(dfiles):
+    """
+    Produce the datapackage json for the eia860m archival collection.
+
+    Args:
+        metadata: dict of fixed metadata descriptors
+
+    Returns:
+        dict: fields suited to the frictionless datapackage spec
+        https://frictionlessdata.io/specs/data-package/
```
Comment (zaneselvans) [inline frictionless/eia860m_raw.py:47]: What's up with the `dfiles` argument, but `metadata` in the docstring Args section?

File: zs/metadata.py [hunk 2]
```diff
@@ -52,6 +52,30 @@
     ]
 }
 
+# Unaltered Eia860 archive.
```
Comment (zaneselvans) [inline zs/metadata.py:55]: Also eia860m in comment?

**Review by zaneselvans — APPROVED** [review]
Minor docstring & comment mismatches. Otherwise great!

File: frictionless/eia860m_raw.py (hunk 1 above)
Comment (cmgosnell) [reply frictionless/eia860m_raw.py:47]: good point! this was a copy/paste but I was wondering. I fixed it here and also in the other modules.

**Pull request merged**